tests/test_size.py

```diff
@@ -1,23 +1,44 @@
 import ast
 
-from vulture.lines import estimate_lines
+from vulture.lines import count_lines
 
+import pytest
 
-def check_size(example, span):
-    # offset of one because 'node' is ast.Module object
-    node = ast.parse(example)
-    assert estimate_lines(node) - 1 == span
+HAS_ASYNC = getattr(ast, 'AsyncFunctionDef', None)
 
 
-def test_size_function():
+def skip_if_not_has_async(item):
```

Comment:
Could you please give me write access to your branch? I have some minor nitpicks that I'd like to quickly change.